package.json

```diff
@@ -19,7 +19,10 @@
     "email": "support@digitalbazaar.com",
     "url": "http://digitalbazaar.com"
   },
-  "license": "ISC",
+  "license": {
+    "type": "BSD-3-Clause",
+    "url": "https://github.com/veres-one/did-veres-one/blob/master/LICENSE"
+  },
```

Comment:
This style is listed as outdated: https://docs.npmjs.com/files/package.json#license.
Since we're using a standard license, it should just be `"license" : "BSD-3-Clause"`.

Reply:
Err.. nm, I'm reviewing the wrong thing.